### ISSUE TITLE
Add support for other directory structures?

### ISSUE THREAD

**Issue by JamieMason** [FormidableLabs/lank#8]
I'm really interested in `lank` but am working on a project with this structure. I'd like to be able to develop on `consuming-app` with the libraries from `lerna-monorepo` linked using `lank`.

```
├── consuming-app
└── lerna-monorepo
    └── packages
        ├── lib-1
        ├── lib-2
        └── lib-3
```

I think `lank` is currently set to work against this structure?

```
├── consuming-app
├── lib-1
├── lib-2
└── lib-3
```

It _might_ be possible to add support for custom project locations without a breaking change if https://github.com/FormidableLabs/lank/blob/88c3f459d9b844986c6e8f8a1519fdab88571a10/lib/config.js#L68 were to use `path.resolve` instead of `path.join`? 

If the current config values of module names are present, then the current behaviour could be used with the `siblingPath` etc but if a relative or absolute path is set instead, then could that be used to locate the project to be linked? Some more refactoring would be needed of course than just a change to this line.

Please let me know if this idea seems worthwhile and aligns with the aims of the project and I'd be happy to help if needed. Feel free to disregard the suggested approach as I don't know this codebase well.

Thanks.

**Comment by ryan-roemer** [comment]
Yep, lank currently just sets one `NODE_PATH` addition / infers one project root directory. We've had some other folks asking for multiple root support to better handle monorepos. I'm a bit crazy busy right now, but will try and get out the feature in the coming weeks...

**Comment by JamieMason** [comment]
No rush at all Ryan, thanks for this great util. If I can help please let me know.
